bin/check-rabbitmq-queues-synchronised.rb

```diff
@@ -64,7 +70,8 @@ def get_queues(config)
     url_prefix = config[:ssl] ? 'https' : 'http'
     options = {
       user: config[:username],
-      password: config[:password]
+      password: config[:password],
+      verify_ssl: !config[:verify_ssl_off]
```

Comment:
this is part of why I'd rather just use a positive name rather than a negative name.